### ISSUE TITLE
'node' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by AHMED-GAMAL-AG** [beyondcode/herd-community#649]
### Platform

Windows

### Operating system version

windows 11

### System architecture

Windows

### Herd Version

1.0.1

### PHP Version

8.3

### Bug description

node is installed successfully through Herd, but it is not recognized in the terminal. Am i doing something wrong ?

![image](https://github.com/beyondcode/herd-community/assets/76778937/d5557fab-42e0-43b7-b78a-8ecd5bb0bfcf)

![image](https://github.com/beyondcode/herd-community/assets/76778937/95dd5fa2-3da1-409b-9868-62f47093f6c4)

**Comment by mpociot** [comment]
Can you verify that Herd is in your PATH?
Also, did you try and open a new terminal tab just to be sure?

**Comment by AHMED-GAMAL-AG** [comment]
sorry for late reply Here is what I found in the path. Yes, I opened a new terminal path.

![image](https://github.com/beyondcode/herd-community/assets/76778937/2b81d98c-fddf-4298-8016-d3fdaaa1c805)

**Comment by mpociot** [comment]
This is pretty weird. Do you also have the `NVM_HOME` and `NVM_SYMLINK`  environment variables set?

What do you get when your run `nvm ls` in your terminal?

**Comment by AHMED-GAMAL-AG** [comment]
I get this when running `nvm ls` my terminal

![image](https://github.com/beyondcode/herd-community/assets/76778937/2d2e9200-c036-461c-b0bb-92a9bdbbcf8c)

and the variables are set in the enviroment

![image](https://github.com/beyondcode/herd-community/assets/76778937/71e35e6e-76ac-4198-9110-ed7212762ef5)

**Comment by AHMED-GAMAL-AG** [comment]
The issue was that node wasn't completely uninstalled from my system after I removed it with the Windows uninstaller 

to fix it, follow these instructions [https://www.geeksforgeeks.org/how-to-completely-remove-node-js-from-windows/](URL)

**Comment by adevade** [comment]
I had some trouble with this as well, and found out that the value from `NVM_SYMLINK` was missing from my system `PATH`. Not sure if it got removed when I uninstalled the old Node.js (which I did after installing Herd), or if I removed it when I was cleaning up and troubleshooting. However, manually adding `C:\Program Files\nodejs` to my system `PATH` fixed it. 👍

Just thought I'd leave this here for future reference. ✌️
